### ISSUE TITLE
Some skinning feature requests

### ISSUE THREAD

**Issue by kathelynn** [ppy/osu#11522]
- Per-mode skinning to change skins between different modes
- Beatmap skinning, if this ever comes as a new meta, let us override some areas such as the hit objects to a selected skin, or the health bar positioning _(as long as it intelligently decides the position and size for smaller screens on different beatmaps)_ for example. 
- [osu skin randomizer](https://osu.ppy.sh/community/forums/topics/777098) as a fun mod that randomizes gameplay skin elements. Probably. Make it do cool stuff as part of osu lazer being able to switch skins in-game? I don't know, sounds cool.
- Skin editor and beatmap editor would be nice to be integrated together.
- In addition, it would be cool if the beatmap could recognize it's own skin and then do stuff like have it blend with the storyboard.
- If menu skinning ever happens, it would be nice to have most common elements like the pause/retry screen button positions be changed.

I don't have designs up to some of them for now but I will do prototyping if needed & I get the chance. But I really support the push for custom skinning and hope that it does more given the potential. I hope one day I can see a skin accompanying a beatmap sometime soon and for people to start using it more, it would be pretty cool. I loved some of them that did.

**Comment by bdach** [comment]
The suggestions are appreciated, but we don't usually accept "grab-bag" issues like this one because they're too wide in scope. The basics (#4924) have to happen first before any of this can be considered, and might cover some of your concerns (per-mode/menu skinning).
